Dockerfile

```diff
@@ -4,6 +4,7 @@ FROM node:18-alpine AS production
 #RUN apt-get upgrade -y
 
 ARG APP_ENV
+ARG NEW_RELIC_LICENSE_KEY
```

Comment:
Add this as an env similar to `APP_ENV` on line 19.

Reply:
good catch